<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Two_Spectrum_Window</class>
 <widget class="QMainWindow" name="Two_Spectrum_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="Spectrum1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>841</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Spectrum2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>841</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Choose1stSignal">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose first signal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlotTwoSpectrum">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>420</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Two Spectrum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Choose2ndSignal">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Chhose second signal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SecondSignalPath">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>160</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="FirstSignalPath">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate1stSpectrum">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>320</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute first spectrum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate2ndSpectrum">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>370</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute second spectrum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculateTwoSpectra">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>270</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute two spectra</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
